.vscode/settings.json

```diff
@@ -15,5 +15,8 @@
   "npm.packageManager": "yarn",
   "editor.formatOnSave": true,
   "editor.defaultFormatter": "dbaeumer.vscode-eslint",
-  "eslint.format.enable": true
+  "eslint.format.enable": true,
+  "[typescript]": {
```

Comment:
fyi @rorolopetegui this workspace setting is necessary for me to override the formatter I had for default for TypeScript